_includes/specials/taproot/en/10-ln-with-taproot.md

```diff
@@ -0,0 +1,198 @@
+*By [ZmnSCPxj][]*
```

Comment:
It might be nice here or early in this segment to note what ZmnSCPxj's role in the efforts outlined below are as it provides better context for later in the segments where estimates are being given ("what who is this person to be estimating?")

Reply:
Good suggestion! I forgot that, outside of LN dev, there are only a few who know how epic ZmnSCPxj is.